BookingsApi/BookingsApi.Contract/Requests/UpdateHearingParticipantsRequest.cs

```diff
@@ -0,0 +1,28 @@
+using System;
+using System.Collections.Generic;
+
+namespace BookingsApi.Contract.Requests
+{
+    public class UpdateHearingParticipantsRequest
+    {
+        /// <summary>
+        ///     List of new participants
+        /// </summary>
+        public List<ParticipantRequest> NewParticipants { get; set; } = new List<ParticipantRequest>();
```

Comment:
Should we be initialising the Lists here?

Reply:
Lists should always be initialised and never left null. It protects against null reference exceptions and also makes sense conceptually.

Reply:
It mostly caught my eye because it's different to other places in the code where we do things like set it in the constructor. This does look cleaner in my eyes just I couldn't find details online about the best ways to do it.